ui/pages/create-account/new-account.component.js

```diff
@@ -43,21 +50,39 @@ export default class NewAccountCreateForm extends Component {
         });
     };
 
+    const accountNameExists = (allAccounts, accountName) => {
+      const accountsNames = allAccounts.map((item) => item.name);
+
+      return accountsNames.includes(accountName);
+    };
+
     return (
       <div className="new-account-create-form">
         <div className="new-account-create-form__input-label">
           {this.context.t('accountName')}
         </div>
         <div>
           <input
-            className="new-account-create-form__input"
+            className={classnames('new-account-create-form__input', {
+              'new-account-create-form__input__error': accountNameExists(
```

Comment:
Nit: Rather than repeat `accountNameExists(accounts, newAccountName)` three times, it would be easier to read this code if it was called once and the result stored in a variable.

Reply:
Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3